example

```diff
@@ -295,10 +305,10 @@ def start_helper(args, fd=None, socket=None):
         raise ValueError("fd or socket required")
 
     cmd.append(f"--interface-id={interface_id}")
+    cmd.append(f"--operation-mode={args.operation_mode}")
```

Comment:
This is likely wrong when using "host", since the helper always set start/end ip addresss and subnet mask, which may be wrong when using host mode. This should be fixed in the helper.